syntax="proto3";

package transport_proto;

message RouteSettings {
	int32 bus_wait_time = 1;
	double bus_velocity = 2;
}

message EdgeId {
	uint32 has_value = 1;
	uint32 edge_id = 2;
}

message RouteInternalData {
	uint32 has_value = 1;
	double weight = 2;
	EdgeId edge_id = 3;
}

message RouteInternalDataArr {
	repeated RouteInternalData data = 1;
}

message Router {
	repeated RouteInternalDataArr data = 1;
}